terminology.md

```diff
@@ -313,6 +313,27 @@ To make something a first-class concept in a language specification, such that i
 #### References
 - [Reification](https://en.wikipedia.org/wiki/Reification_(computer_science))
 
+### Handler
+
+#### Definition
+
+A parameter value that is a JavaScript object with properties looked up lazily. It is used to configure behavior of a function. Methods on the options bag object should be invoked with the handler as the default value for `this`. It is expected that this parameter has specific well known properties in their structure. Handlers are expected to potentially be extended over time.
```

Comment:
```suggestion
A parameter value that is a JavaScript object with properties looked up lazily (used in `new Proxy`). It is used to configure behavior of a function. Methods on the options bag object should be invoked with the handler as the default value for `this`. It is expected that this parameter has specific well known properties in their structure. Handlers are expected to potentially be extended over time.
```

Reply:
this is not unique to proxy if we consider DOM. I also dont think we should carve out that it only applies to proxies. future APIs may adopt this pattern

Reply:
sure, my suggestion doesn't indicate it's unique to proxy, just that it *is* used in Proxy, as an example.

Where in the DOM is this pattern used?

Reply:
Various places in DOM, easy to see for `.addEventListener`

```mjs
// never even returns a function to invoke
window.addEventListener('foo', {
  UNIQUE_NAME: true,
  get handleEvent() { console.log(this); }
});
window.dispatchEvent(new CustomEvent('foo'));
```

Reply:
wow, i have never seen *any* code pass anything but a function in that position of `addEventListener`.